azure-pipelines.yml

```diff
@@ -144,6 +144,14 @@ jobs:
       continueOnError: true
       condition: not(succeeded())
 
+    - task: PublishBuildArtifacts@1
+      displayName: NuPkg
+      inputs:
+        PathtoPublish: '$(Build.SourcesDirectory)\artifacts\packages\Release\PreRelease'
+        ArtifactName: 'NuPkg'
```

Comment:
Nit: I'd call it `Packages - PreRelease`. We publish all packages as `Packages` in official build, so to have some consistency.